chord_metadata_service/chord/urls.py

```diff
@@ -20,13 +21,22 @@
     path('data-types/<str:data_type>/metadata_schema', views_data_types.data_type_metadata_schema,
          name="data-type-metadata-schema"),
 
-    path('tables', views_search.table_list, name="chord-table-list"),
-    path('tables/<str:table_id>', views_search.table_detail, name="chord-table-detail"),
-    path('tables/<str:table_id>/summary', views_search.chord_table_summary, name="table-summary"),
-    path('tables/<str:table_id>/search', views_search.chord_public_table_search, name="public-table-search"),
     path('search', views_search.chord_search, name="search"),
     path('fhir-search', views_search.fhir_public_search, name="fhir-search"),
     path('private/fhir-search', views_search.fhir_private_search, name="fhir-private-search"),
     path('private/search', views_search.chord_private_search, name="private-search"),
-    path('private/tables/<str:table_id>/search', views_search.chord_private_table_search, name="private-table-search"),
+
+    path('datasets', DatasetViewSet.as_view({'get': 'list'}), name="chord-dataset-list"),
+    path('datasets/<str:dataset_id>', DatasetViewSet.as_view({'get': 'retrieve', 'delete': 'destroy'}),
+         name="chord-dataset-detail"),
+    path('datasets/<str:dataset_id>/summary', views_search.dataset_summary, name="chord-dataset-summary"),
+    path('datasets/<str:dataset_id>/data-types/<str:data_type>', views_data_types.dataset_data_type,
+         name="chord-dataset-data-type"),
+
+    path('datasets/<str:dataset_id>/data-types', views_data_types.dataset_datatype_summary,
+         name="chord-dataset-data-type-summary"),
+
+    path('datasets/<str:dataset_id>/search', views_search.public_dataset_search, name="public-dataset-search"),
+    path('private/datasets/<str:dataset_id>/search', views_search.private_dataset_search,
```

Comment:
would be nice to deprecate dataset search vs private/datasets search (although maybe thats a v14 thing)

Reply:
also how does this search actually work, given that we dont have one data type per dataset?

Reply:
The data_type is given by a query parameter extracted by  `views_searh.get_chord_search_parameters`.